visualdl/server/tsne.py

```diff
@@ -0,0 +1,184 @@
+#
```

Comment:
We should give the original author a citation 😀 

`https://github.com/bhauman/neurpy/blob/master/tsne.py`

Reply:
I did give the author credit, I left my comments under the author's comments.